src/models/participant/facade.js

```diff
@@ -351,7 +351,7 @@ const getParticipantLimitByParticipantCurrencyLimit = async (participantId, curr
           .innerJoin('participantCurrency AS pc', 'pc.participantId', 'participant.participantId')
           .innerJoin('participantLimit AS pl', 'pl.participantCurrencyId', 'pc.participantCurrencyId')
           .select(
-            'participant.participantID AS participantId',
+            'participant.participantId AS participantId',
```

Comment:
how does this work? I thought mysql was case sensitive.

Reply:
I wonder how the previous was working actually because of the same reason...

Reply:
Hmm. If we are wrong here, where will this error be picked up? In end to end tests?